tracee-ebpf/tracee/tracee.go

```diff
@@ -1108,6 +1152,29 @@ func (t *Tracee) processEvent(ctx *context, args map[argTag]interface{}) error {
 	return nil
 }
 
+func getFileHash(fileName string) string {
+	f, _ := os.Open(fileName)
+	if f != nil {
+		defer f.Close()
+		h := sha256.New()
+		_, _ = io.Copy(h, f)
+		return hex.EncodeToString(h.Sum(nil))
+	}
+	return ""
+}
+
+func (t *Tracee) updateProfile(sourceFilePath string) {
+	t.profiledFiles[sourceFilePath] = profilerInfo{}
```

Comment:
What if the same file was written twice, but with different content? We will only capture the last instance of written file, won't we? By adding the ctime to the key of the map, we can save all of the relevant instances

Reply:
`sourceFilePath` does include the ctime after https://github.com/aquasecurity/tracee/pull/725#discussion_r636348145 
(see L1131). it's just a bad var name (let's rename it)